utils.js

```diff
@@ -264,6 +264,36 @@ export function getAllFilesWithIgnore(dirPath, pattern, ignoreList) {
   }
 }
 
+/**
+ * Method to return sha1 hash of a file
+ *
+ * @param {string} filePath
+ * @returns {string} hash.digest("hex")
+ */
+function calculateHash(filePath) {
```

Comment:
There is already a function to compute checksum

https://github.com/CycloneDX/cdxgen/blob/068063aad71099eef383e0aeb35e88c001793906/utils.js#L7754